src/common/vitals/largest-contentful-paint.js

```diff
@@ -7,22 +7,26 @@ import { cleanURL } from '../url/clean-url'
 export const largestContentfulPaint = new VitalMetric(VITAL_NAMES.LARGEST_CONTENTFUL_PAINT)
 
 if (isBrowserScope) {
-  onLCP(({ value, entries }) => {
+  onLCP(({ value, attribution }) => {
   /* Largest Contentful Paint - As of WV v3, it still imperfectly tries to detect document vis state asap and isn't supposed to report if page starts hidden. */
     if (initiallyHidden || largestContentfulPaint.isValid) return
 
-    const lcpEntry = entries[entries.length - 1] // this looks weird if we only expect one, but this is how cwv-attribution gets it so to be sure...
-    largestContentfulPaint.update({
-      value,
-      entries,
```

Comment:
This is changing the signature of the object we report out via the update method. Where does this change need to be accounted for and has it been?